core/src/test/java/io/questdb/test/cutlass/http/line/AbstractLineHttpFuzzTest.java

```diff
@@ -193,6 +193,8 @@ public void setUp() {
     public void setUp2() {
         long s0 = System.currentTimeMillis();
         long s1 = System.nanoTime();
+        s0 = 1706810999286L;
+        s1 = 1057592012839L;
```

Comment:
Are these left-overs that need to be reverted?

Reply:
yes 😊

Reply:
thank you!